src/services/side-panels/ui/toggel.scss

```diff
@@ -0,0 +1,8 @@
+.side-panel {
+  transition: all;
+  visibility: hidden;
+}
+
+.active {
+  visibility: visible;
```

Comment:
transition: none (not sure this is the syntax)

Reply:
[this is the syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility)